Servers/controllers/project.ctrl.ts

```
@@ -145,12 +140,16 @@ export async function createProject(req: Request, res: Response): Promise<any> {
           STATUS_CODE[400]({ message: "project_title and owner are required" })
         );
     }
+    console.log(newProject);
 
     const createdProject = await createNewProjectQuery(newProject);
-    const assessments = await createNewAssessmentQuery({
+    const assessments: Object = await createNewAssessmentQuery({
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)